Tools/clinic/clinic.py

```diff
@@ -2112,6 +2112,8 @@ def parse(self, input):
                          traceback.format_exc().rstrip())
             printer.print_block(block)
 
+        clinic_out = []
```

Comment:
Re https://github.com/python/cpython/pull/104152/files#r1184646280

```suggestion
        clinic_out: list[str, str] = []
```